requirements.txt

```diff
@@ -1,20 +1,5 @@
-ashes==17.0.0
-attrs==18.1.0
-boltons==18.0.1
-chert==18.0.0
-glom==18.1.0
-html5lib==1.0.1
-hyperlink==18.0.0
-idna==2.8
-lithoxyl==0.4.3
-Markdown==2.6.2
-pluggy==0.4.0
-py==1.4.33
-Pygments==1.6
-python-dateutil==2.7.5
-PyYAML==3.13
-schema==0.6.8
-six==1.12.0
-tox==2.7.0
-virtualenv==15.1.0
-webencodings==0.5.1
+boltons==24.1.0
```

Comment:
Just noticing this file got a big haircut. I'm curious: how did you get this newer, shorter requirements.txt?

Reply:
It's not complete. I didn't add back the chert dependencies yet. Can you release the newest version of the chert pip package?